src/playlist-loader.js

```diff
@@ -10,6 +10,27 @@ import { mergeOptions, EventTarget, log } from 'video.js';
 import m3u8 from 'm3u8-parser';
```

Comment:
I think it makes sense, at some point, to rename some files here.  Given this is now a DASH/HLS project it might make sense to rename this to something like `hls-playlist-loader` (or something similar) to make it clear that there isn't one playlist to rule them all (or that one is not a subset of the other).